src/NLog.Web.AspNetCore/README.md

```diff
@@ -17,3 +17,11 @@ Useful Links
 - [Getting started with ASP.NET Core 3](https://github.com/NLog/NLog/wiki/Getting-started-with-ASP.NET-Core-3)
 - [ASP.NET Core Layout Renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore)
 - [Logging Troubleshooting](https://github.com/NLog/NLog/wiki/Logging-troubleshooting)
+
+Registration of NLog.Web in the NLog.config File
```

Comment:
Think the "Useful links" should always be at the bottom. Any advice should be move aboe.

Reply:
complete